qwik-graphql-tailwind/src/routes/[owner]/[name]/index.tsx

```diff
@@ -77,9 +77,9 @@ export default component$(() => {
 
   const { path, name, owner } = useLocation().params;
 
-  const _owner = owner.replace(SPECIAL_PERIOD_CHAR_URL_ENCODED_REGEX, '.');
-  const _name = name.replace(SPECIAL_PERIOD_CHAR_URL_ENCODED_REGEX, '.');
-  const _path = path?.replace(SPECIAL_PERIOD_CHAR_URL_ENCODED_REGEX, '.') || '';
+  const _owner = owner && owner.replace(SPECIAL_PERIOD_CHAR_URL_ENCODED_REGEX, '.');
+  const _name = name && name.replace(SPECIAL_PERIOD_CHAR_URL_ENCODED_REGEX, '.');
+  const _path = (path && path?.replace(SPECIAL_PERIOD_CHAR_URL_ENCODED_REGEX, '.')) || '';
 
```

Comment:
could have used `?`, won't it have work?

Reply:
you were right, thanks @AllStackDev1